hooks/pre_gen_project.py

```diff
@@ -0,0 +1,25 @@
+"""Python module name validation."""
+import logging
+import sys
+import re
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+
+def validate_python_module_name():
+    """Check that the module name given by the user is an acceptable format."""
+    module_name = "{{ cookiecutter.module_name }}"
+    # The following regex will only match valid Python module names
+    if not re.match(r"^[a-z][_a-z0-9]+$", module_name):
+        # Warn the user, but allow for override
```

Comment:
How can the user override? This seems like it's just a hard system exit? I actually prefer that the user DON'T get to override -- they have to put in a valid python module name. There's only pain if they don't.

Reply:
This is a dead comment from when I first wrote this up.

Reply:
Cool, can you remove?

Reply:
Yep removing